models/demos/ttnn_resnet/tests/test_perf_ttnn_resnet.py

```diff
@@ -36,7 +36,7 @@
     use_signpost = False
 
 # TODO: Create ttnn apis for this
-ttnn.dump_device_profiler = ttnn.experimental.device.DumpDeviceProfiler
```

Comment:
Can you get rid of this and replace the `ttnn.dump_device_profiler` with `ttnn.DumpDeviceProfiler` in the file?

Reply:
Sure @tt-aho